client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -78,7 +84,8 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goNext, goToStep, flow }: S
 	const startWritingFlowTasks: Task[] | null = getArrayOfFilteredTasks(
 		tasks,
 		flow,
-		isEmailVerified
+		isEmailVerified,
+		hasGoalPaidSubscribers
```

Comment:
Highlighting that this `hasGoalPaidSubscribers` ended up here in `startWritingFlowTasks` (writing flow, not newsletter flow) in a rebase with `trunk` and isn't working for Newsletter anymore. We need to move this to backend API as I noted in https://github.com/Automattic/wp-calypso/pull/76386#issuecomment-1529966672